redhat-cop/casl-ansible
# Additional DNS clean-up and preparing for release cut of contrib v3.5.0

#### What does this PR do?
Updating galaxy to consume the v3.5.0 release of contrib + changing the DNS processing to use the roles rather than ad-hoc playbooks.

#### How should this be manually tested?
Run through the install steps as outlined in the README and validate that the install works as it should.

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/casl
